README.md

```diff
@@ -26,14 +18,35 @@ Conjur provides:
 
 * try Conjur: [Start Here](https://try.conjur.org)
 * [support](https://try.conjur.org/support.html)
-* [API Documentation][api-doc]
+* [API Documentation][api]
+
+[![Join Conjur Slack](https://img.shields.io/badge/slack-open-e01563.svg)][slack]
+[![Conjur API Ruby Gem](https://img.shields.io/gem/dt/conjur-api.svg?label=ruby%20gem%20downloads)][gem]
+[![Docker Pulls](https://img.shields.io/docker/pulls/cyberark/conjur.svg)][docker]
+[![Issues ready for work](https://img.shields.io/waffle/label/cyberark/conjur/ready.svg?label=issues%20ready%20for%20work)][waffle]
+[![Issues in progress](https://img.shields.io/waffle/label/cyberark/conjur/in%20progress.svg)][waffle]
+[![Follow Conjur on Twitter](https://img.shields.io/twitter/follow/conjurinc.svg?style=social&label=Follow%20%40ConjurInc)][twitter]
+
+
+#### Conjur module on PuppetForge
```

Comment:
all these shields should go in the README for https://github.com/cyberark/conjur-puppet.

Puppet will have an integration page/section and tutorial on conjur.org soon.

We should also link the puppet repo to those pages once they exist.

Reply:
Okay, I'll create a PR there for the badges